src/screens/SelectRecipient/index.js

```diff
@@ -95,7 +96,9 @@ class SelectRecipient extends Component<Props, State> {
                 console.log("scan qr code"); // eslint-disable-line no-console
               }}
             >
-              <Icon name="qrcode" size={16} color={colors.live} />
+              <Text>
+                <Icon name="qrcode" size={16} color={colors.live} />
+              </Text>
```

Comment:
any reason to put that in a Text?

Reply:
Yes, it crashes because Icon is a font base icon and uses text to render, so it breaks iOS otherwise (didn't test on android without the <Text>)